Patch 1

eaton-lab opened the pull request:
added setuptools to host section; updated version to 2.0; added eaton-lab as a maintainer.

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

@conda-forge-admin, please rerender

regro-cf-autotick-bot committed:
updated v2.0.0

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-build 24.7.1, conda-smithy 3.40.1, and conda-forge-pinning 2024.09.24.01.00.11

eaton-lab committed:
Update meta.yaml

added setuptools to host section

eaton-lab committed:
Update meta.yaml

added eaton-lab as a maintainer. Updated version to 2.0.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - For the `run` section of recipe, you should usually use `python >={{ python_min }}` for the `python` entry.
   - For the `test.requires` section of recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).
 * ℹ️ PyPI default URL is now pypi.org, and not pypi.io. You may want to update the default source url.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/13998679859](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13998679859). Examine the logs at this URL for more detail.</sub>

Commit:
MNT: Re-rendered with conda-build 25.1.2, conda-smithy 3.47.0, and conda-forge-pinning 2025.03.21.13.13.18

eaton-lab committed:
Update meta.yaml

added usage of `{{ python_min }}` as suggested by bot, and changed pypy.io to pypy.org.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ ``requirements: run: python >= 9999`` should not contain a space between relational operator and the version, i.e. ``python >=9999``

For **recipe/meta.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `run` section of recipe, you should usually use `python >={{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).
 * ℹ️ top-level output has some malformed specs:
- In section run: `python >= 9999
Requirement spec fields should always be space-separated to avoid known issues in conda-build. For example, instead of `name =version=build`, use `name version.* build`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/13998736066](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13998736066). Examine the logs at this URL for more detail.</sub>

eaton-lab committed:
Update meta.yaml

removed space between `>=` and `{{ python_min }}`

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

eaton-lab committed:
Update linux_64_.yaml

Added the `python_min` variable set to default '3.9'

eaton-lab committed:
Update meta.yaml

Oh, I see that setuptools was replaced with flit in toyplot 2.0. Now meta.yaml requires flit-core in build and host.

eaton-lab commented:
Hi @dpryan79 , please review this patch. I am happy to help review this repo in the future to keep the toyplot recipe up to date.

eaton-lab commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14035978928](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14035978928). Examine the logs at this URL for more detail.</sub>

eaton-lab committed:
Merge branch '2.0.0_h2981ff' into patch-1

eaton-lab commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14137893563](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14137893563). Examine the logs at this URL for more detail.</sub>

h-vetinari commented on recipe/meta.yaml:
```diff
@@ -1,34 +1,41 @@
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
+  build:
+    - python {{ python_min }}
+    - pip
+    - flit-core
```
this section should not be necessary, please remove it.

eaton-lab committed:
Update meta.yaml

removed requirements build section

h-vetinari reviewed (approved):
Thanks for stepping up! 🙏
